folly/experimental/symbolizer/Dwarf.cpp

```diff
@@ -19,7 +19,11 @@
 
 #include <type_traits>
 
-#include <dwarf.h>
+#if FOLLY_HAVE_LIBDWARF_DWARF_H
+# include <libdwarf/dwarf.h>
```

Comment:
Any reason we can't just add `/usr/include/libdwarf/` as an extra include path? That would mean these macro checks aren't needed.